velox/exec/tests/MergeJoinTest.cpp

```diff
@@ -592,6 +592,38 @@ TEST_F(MergeJoinTest, semiJoin) {
       core::JoinType::kRightSemiFilter);
 }
 
+TEST_F(MergeJoinTest, rightJoin) {
+  auto left = makeRowVector(
+      {"t0"},
+      {makeNullableFlatVector<int64_t>(
+          {1, 2, std::nullopt, 5, 6, std::nullopt})});
+
+  auto right = makeRowVector(
+      {"u0"},
+      {makeNullableFlatVector<int64_t>(
+          {1, 5, 6, 8, std::nullopt, std::nullopt})});
+
+  createDuckDbTable("t", {left});
+  createDuckDbTable("u", {right});
+
+  // Right join.
+  auto planNodeIdGenerator = std::make_shared<core::PlanNodeIdGenerator>();
```

Comment:
please also make sure we reuse the existing unit tests for left joins with filters. There are super complicated to get right.

Reply:
@pedroerp Adding the related right join unit tests in leftJoinFilters tests.